src/server/conn.rs

```diff
@@ -615,6 +615,7 @@ where
     S::Error: Into<Box<dyn StdError + Send + Sync>>,
     I: AsyncRead + AsyncWrite + Unpin,
     B: HttpBody + 'static,
+    B::Data: Send,
```

Comment:
Is there something that was already requiring this bound? If not, I think it could be a breaking change, no?

Reply:
I'll carefully compare the rustdoc outputs before and after that change, that's why I made it a separate commit in the first place heh.

Reply:
This is indeed a breaking change. I can limit its scope to the two methods `poll_without_shutdown` and `without_shutdown`.